CRM/Financial/BAO/FinancialAccount.php

```diff
@@ -395,7 +395,7 @@ public static function checkFinancialTypeHasDeferred($params, $contributionID =
         $params, $items);
       }
       else {
-        $items = CRM_Price_BAO_LineItem::getLineItems($contributionID, 'contribution', TRUE, TRUE, TRUE);
+        $items = CRM_Price_BAO_LineItem::getLineItemsByContributionID($contributionID, TRUE);
```

Comment:
Actually you are probably the best person to review this function & see if we can avoid referring to $isQuick from it